src/photosui.cs

```diff
@@ -138,6 +137,10 @@ interface PHLivePhotoViewDelegate {
 
 		[Export ("livePhotoView:didEndPlaybackWithStyle:")]
 		void DidEndPlayback (PHLivePhotoView livePhotoView, PHLivePhotoViewPlaybackStyle playbackStyle);
+
+		[TV (10,0), iOS (9,1), NoMac]
+		[Export ("livePhotoView:extraMinimumTouchDurationForTouch:withStyle:")]
+		double ExtraMinimumTouchDuration (PHLivePhotoView livePhotoView, UITouch touch, PHLivePhotoViewPlaybackStyle playbackStyle);
```

Comment:
Maybe verb-ify this one

Reply:
Suggestions welcome on a verb here. You are providing the extra min touch duration. 

`ProvideExtraMinimumTouchDuration` was the best I could come up with, but that's a large delta and I didn't love that.

Reply:
I'm not sure I understand why `GetExtraMinimumTouchDuration` doesn't work?